var/spack/repos/builtin/packages/hepmc/package.py

```diff
@@ -26,12 +26,24 @@ class Hepmc(CMakePackage):
     version('2.06.06', sha256='8cdff26c10783ed4248220a84a43b7e1f9b59cc2c9a29bd634d024ca469db125')
     version('2.06.05', sha256='4c411077cc97522c03b74f973264b8d9fd2b6ccec0efc7ceced2645371c73618')
 
-    depends_on('cmake@2.6:', type='build')
+    variant('python', default=False, description='Enable Python bindings')
+    variant('rootio', default=False, description='Enable ROOT I/O')
+
+    depends_on('cmake@2.8.9:', type='build')
+    # FIXME: Officially supports Python3, but the build system doesn't find it
+    depends_on('python@:2.99.99', when='+python')
```

Comment:
```suggestion
    depends_on('python@:2.99.99', when='+python', type=('build', 'run'))
```
Presumably?

Reply:
Wouldn't that lead to a CMake configuration failure?

Reply:
What do you mean? Does it link to the Python libraries, or just run the binary?

Reply:
It builds Python bindings, so it links to the Python libraries.